ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -456,6 +469,15 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
     workflow: openshift-e2e-aws-sdn
+- as: e2e-aws-ovn-multi
```

Comment:
Sippy's expecting multiRegex = regexp.MustCompile(`(?i)-heterogeneous`)

Can we rename so it picks up the right variant there?

Or is this a multi-arch payload, but regular amd64 job?

Reply:
It is still an amd64 job with multiarch payload

Reply:
multiRegex are used for true heterogeneous deployments only (cp=amd64, nodes=amd64+ arm64)